lib/puppet/provider/archive/wget.rb

```diff
@@ -0,0 +1,42 @@
+Puppet::Type.type(:archive).provide(:wget, :parent => :ruby ) do
+
+  commands   :wget => '/usr/bin/wget'
+  defaultfor :operatingsystem => :linux
```

Comment:
I used puppet --debug and saw the command. Also used the debugger to actually step though the code. Anyway.. I'll change it to:
```ruby
defaultfor :feature => :posix
```

or